tests/buyers/views/test_buyers.py

```diff
@@ -1969,6 +1969,14 @@ def setup_method(self, method):
                 "blah": [False, True]
             }
         ]
+        self.bad_response = {
+            "supplierName": "Tea Boy Ltd.",
+            "respondToEmailAddress": "teaboy@example.com",
+            "availability": "Tomorrow",
+            "essentialRequirementsMet": False,
```

Comment:
In our flow, if you try to answer no to essential requirements met, we don't actually save your answer to the brief response. So we would actually never expect essential requirements met to be set and be False.